core/src/main/java/org/apache/carbondata/core/datamap/IndexChooser.java

```diff
@@ -59,76 +59,76 @@
  *   the datamap which has fewer columns that is the first datamap.
  */
 @InterfaceAudience.Internal
-public class DataMapChooser {
+public class IndexChooser {
 
   private CarbonTable carbonTable;
-  private List<TableDataMap> cgDataMaps;
-  private List<TableDataMap> fgDataMaps;
+  private List<TableIndex> cgIndexes;
+  private List<TableIndex> fgIndexes;
 
-  public DataMapChooser(CarbonTable carbonTable) throws IOException {
+  public IndexChooser(CarbonTable carbonTable) throws IOException {
     this.carbonTable = carbonTable;
-    // read all datamaps for this table and populate CG and FG datamap list
-    List<TableDataMap> visibleDataMaps =
-        DataMapStoreManager.getInstance().getAllVisibleDataMap(carbonTable);
+    // read all indexes for this table and populate CG and FG index list
+    List<TableIndex> visibleIndexes =
+        DataMapStoreManager.getInstance().getAllVisibleIndexes(carbonTable);
```

Comment:
DataMapStoreManager need to be changed to IndexStoreManager ?

Reply:
Yes, it will be changed in a new pr.